website/search_migration/migrate.py

```diff
@@ -130,6 +130,13 @@ def migrate_users(index, delete, increment=10000):
             es_args={'raise_on_error': False})  # ignore 404s
         logger.info('{} users marked deleted'.format(total_users))
 
+def migrate_collected_guid_metadata(index, delete):
+    cgms = CollectedGuidMetadata.objects.all()
+    if delete:
```

Comment:
Update: there appear to be exactly two cases where `delete` is important -
* if the first branch of the conditional in `set_up_index` is hit -- when the alias is missing or malformed (seemingly should only happen if migrating from nothing, so probably doesn't matter)
* If an old  run of `migrate_search` has failed, leaving the next version of the index partially populated but orphaned (likely). This can be obviated by dropping anything from the "new" index before populating it.